Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla42602.cs

```diff
@@ -7,7 +7,7 @@
 using Xamarin.Forms.Core.UITests;
 #endif
 
-namespace Xamarin.Forms.Controls
+namespace Xamarin.Forms.Controls.Issues
```

Comment:
You'll need to update this namespace in ExportRenderer attribute in Xamarin.Forms.ControlGallery.WindowsUniversal.CustomRenderers.cs.